src/vdaf/prio3.rs

```diff
@@ -416,7 +421,7 @@ where
             let mut deriver = P::init(&helper.joint_rand_param.blind);
             deriver.update(&[aggregator_id]);
             let prng: Prng<T::Field, _> =
-                Prng::from_seed_stream(P::seed_stream(&helper.input_share, b"input share"));
+                Prng::from_seed_stream(P::seed_stream(&helper.input_share, VERS_PRIO3));
```

Comment:
This seems like it regresses in that we no longer have domain separation between PRNGs used for joint rand vs. input sharing vs. prove randomness. Why not do `b"vdaf-00 prio3 input share"` and `b"vdaf-00 prio3 joint rand"` and so on?

Reply:
Whether this is a regresssion depends on how we define security for VDAFs. My intuition is that domain separation for vector type (input share, proof share, query rand, etc.) isn't going to be needed.

That said, a binding to the draft version is useful. My goal is to keep the info string under 16 bytes so that it fits into a single AES block. Note that I will add an additional byte for input share and proof share: see the commit message for https://github.com/cjpatton/vdaf/commit/29840fd017e6dc9103d1fa996475e0ccaa71d222 for an explanation. I will tack this on when I do my pass of your comments.